src/bindings/python/src/openvino/runtime/utils/util.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from openvino._pyopenvino.util import clone_model as clone_model_base
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from openvino.runtime import Model
```

Comment:
Isn't this introducing some circular dependency normally? It only triggers with mypy checks, right?

Reply:
yes, this import is trigger only with mypy checks (another way -- suppress annotation check)